x-pack/plugins/siem/public/containers/hosts/details/index.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getOr } from 'lodash/fp';
+import React from 'react';
+import { Query } from 'react-apollo';
+
+import { GetHostDetailsQuery, HostItem } from '../../../graphql/types';
+import { inputsModel } from '../../../store';
+import { getDefaultFetchPolicy } from '../../helpers';
+import { QueryTemplate, QueryTemplateProps } from '../../query_template';
+
+import { HostDetailsQuery } from './host_details.gql_query';
+
+export { HostsFilter } from '../filter';
```

Comment:
I believe this also duplicates the `HostsFilter` export from `x-pack/plugins/siem/public/containers/hosts/index.tsx`